src/.eslintrc.json

```diff
@@ -28,16 +28,21 @@
     "handleErrors_": "readonly",
     "assertCanAccessHistoricData_": "readonly",
     "assertHasData_": "readonly",
-    "setUserProperty_": "readonly",
     "apiKeyProperty_": "readonly",
     "addApiKey_": "readonly",
+    "removeApiKey_": "readonly",
+    "deleteApiKeyProperty_": "readonly",
     "hasApiKeyProperty_": "readonly",
+    "obfuscateApiKey_": "readonly",
+    "checkApiKeyStillValid_": "readonly",
```

Comment:
why`still` valid? Is there another function that checks the validity and then this function/property? starts to handle/persist the checks?

Reply:
One checks